html/en/beats/journalbeat/7.4/add-cloud-metadata.html
html/en/beats/journalbeat/7.4/add-docker-metadata.html
html/en/beats/journalbeat/7.4/add-fields.html
html/en/beats/journalbeat/7.4/add-host-metadata.html
html/en/beats/journalbeat/7.4/add-kubernetes-metadata.html
html/en/beats/journalbeat/7.4/add-labels.html
html/en/beats/journalbeat/7.4/add-locale.html
html/en/beats/journalbeat/7.4/add-observer-metadata.html
html/en/beats/journalbeat/7.4/add-process-metadata.html
html/en/beats/journalbeat/7.4/add-tags.html
html/en/beats/journalbeat/7.4/bandwidth-throttling.html
html/en/beats/journalbeat/7.4/beats-basic-auth.html
html/en/beats/journalbeat/7.4/beats-system-user.html
html/en/beats/journalbeat/7.4/beats-tls.html
html/en/beats/journalbeat/7.4/command-line-options.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/command-line-options.html
@@ -442,37 +442,41 @@
               </code>
               .
              </dd>
-             <dt>
-              <span class="term">
-               <span class="strong strong">
-                <strong>
-                 <code class="literal">
-                  ilm-policy
-                 </code>
-                </strong>
-               </span>
-              </span>
-             </dt>
-             <dd>
-              Exports the index lifecycle management policy to stdout. You can specify the
-              <code class="literal">
-               --es.version
-              </code>
-              and a
-              <code class="literal">
-               --dir
-              </code>
-              to which the policy should be exported as a file rather than exporting to
-              <code class="literal">
-               stdout
-              </code>
-              .
-             </dd>
             </dl>
            </div>
-           <p>
+           <div class="variablelist">
             <a id="ilm-policy-subcommand">
             </a>
+            <dl class="variablelist">
+             <dt>
+              <span class="term">
+               <span class="strong strong">
+                <strong>
+                 <code class="literal">
+                  ilm-policy
+                 </code>
+                </strong>
+               </span>
+              </span>
+             </dt>
+             <dd>
+              Exports the index lifecycle management policy to stdout. You can specify the
+              <code class="literal">
+               --es.version
+              </code>
+              and a
+              <code class="literal">
+               --dir
+              </code>
+              to which the policy should be exported as a file rather than exporting to
+              <code class="literal">
+               stdout
+              </code>
+              .
+             </dd>
+            </dl>
+           </div>
+           <p>
             <span class="strong strong">
              <strong>
               FLAGS
html/en/beats/journalbeat/7.4/community-id.html
html/en/beats/journalbeat/7.4/configuration-general-options.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/configuration-general-options.html
@@ -220,7 +220,7 @@
            <pre class="programlisting prettyprint lang-sh">journalbeat.registry_file: registry</pre>
           </div>
           <h4>
-           <a id="_literal_backoff_literal_phrase_revisionflag_deleted_revision_5_6_1_use_the_option_under_literal_paths_literal_instead_phrase">
+           <a id="_backoff_use_the_option_under_paths_instead">
            </a>
            <code class="literal">
             backoff
@@ -261,7 +261,7 @@
            .
           </p>
           <h4>
-           <a id="_literal_max_backoff_literal_phrase_revisionflag_deleted_revision_5_6_1_use_the_option_under_literal_paths_literal_instead_phrase">
+           <a id="_max_backoff_use_the_option_under_paths_instead">
            </a>
            <code class="literal">
             max_backoff
@@ -302,7 +302,7 @@
            .
           </p>
           <h4>
-           <a id="_literal_seek_literal_phrase_revisionflag_deleted_revision_5_6_1_use_the_option_under_literal_paths_literal_instead_phrase">
+           <a id="_seek_use_the_option_under_paths_instead">
            </a>
            <code class="literal">
             seek
@@ -343,7 +343,7 @@
            .
           </p>
           <h4>
-           <a id="_literal_include_matches_literal_phrase_revisionflag_deleted_revision_5_6_1_use_the_option_under_literal_paths_literal_instead_phrase">
+           <a id="_include_matches_use_the_option_under_paths_instead">
            </a>
            <code class="literal">
             include_matches
@@ -402,7 +402,7 @@
 tags: ["service-X", "web-tier"]</pre>
           </div>
           <h4>
-           <a id="_literal_name_literal">
+           <a id="_name">
            </a>
            <code class="literal">
             name
@@ -429,7 +429,7 @@
            <pre class="programlisting prettyprint lang-yaml">name: "my-shipper"</pre>
           </div>
           <h4>
-           <a id="_literal_tags_literal">
+           <a id="_tags">
            </a>
            <code class="literal">
             tags
@@ -479,7 +479,7 @@
            <pre class="programlisting prettyprint lang-yaml">fields: {project: "myproject", instance-id: "574734885120952459"}</pre>
           </div>
           <h4>
-           <a id="_literal_fields_under_root_literal">
+           <a id="_fields_under_root">
            </a>
            <code class="literal">
             fields_under_root
@@ -509,7 +509,7 @@
   region: us-east-1</pre>
           </div>
           <h4>
-           <a id="_literal_processors_literal">
+           <a id="_processors">
            </a>
            <code class="literal">
             processors
@@ -531,7 +531,7 @@
            for information about specifying processors in your config.
           </p>
           <h4>
-           <a id="_literal_max_procs_literal">
+           <a id="_max_procs">
            </a>
            <code class="literal">
             max_procs
html/en/beats/journalbeat/7.4/configuration-journalbeat-options.html
html/en/beats/journalbeat/7.4/configuration-logging.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/configuration-logging.html
@@ -229,7 +229,7 @@
            config file:
           </p>
           <h4>
-           <a id="_literal_logging_to_stderr_literal">
+           <a id="_logging_to_stderr">
            </a>
            <code class="literal">
             logging.to_stderr
@@ -246,7 +246,7 @@
            command line option.
           </p>
           <h4>
-           <a id="_literal_logging_to_syslog_literal">
+           <a id="_logging_to_syslog">
            </a>
            <code class="literal">
             logging.to_syslog
@@ -269,7 +269,7 @@
            </div>
           </div>
           <h4>
-           <a id="_literal_logging_to_eventlog_literal">
+           <a id="_logging_to_eventlog">
            </a>
            <code class="literal">
             logging.to_eventlog
@@ -282,7 +282,7 @@
            When true, writes all logging output to the Windows Event Log.
           </p>
           <h4>
-           <a id="_literal_logging_to_files_literal">
+           <a id="_logging_to_files">
            </a>
            <code class="literal">
             logging.to_files
@@ -435,7 +435,7 @@
            also sets the debug log level).
           </p>
           <h4>
-           <a id="_literal_logging_metrics_enabled_literal">
+           <a id="_logging_metrics_enabled">
            </a>
            <code class="literal">
             logging.metrics.enabled
@@ -457,7 +457,7 @@
            Note that we currently offer no backwards compatible guarantees for the internal metrics and for this reason they are also not documented.
           </p>
           <h4>
-           <a id="_literal_logging_metrics_period_literal">
+           <a id="_logging_metrics_period">
            </a>
            <code class="literal">
             logging.metrics.period
@@ -470,7 +470,7 @@
            The period after which to log the internal metrics. The default is 30s.
           </p>
           <h4>
-           <a id="_literal_logging_files_path_literal">
+           <a id="_logging_files_path">
            </a>
            <code class="literal">
             logging.files.path
@@ -487,7 +487,7 @@
            section for details.
           </p>
           <h4>
-           <a id="_literal_logging_files_name_literal">
+           <a id="_logging_files_name">
            </a>
            <code class="literal">
             logging.files.name
@@ -506,7 +506,7 @@
            .
           </p>
           <h4>
-           <a id="_literal_logging_files_rotateeverybytes_literal">
+           <a id="_logging_files_rotateeverybytes">
            </a>
            <code class="literal">
             logging.files.rotateeverybytes
@@ -519,7 +519,7 @@
            The maximum size of a log file. If the limit is reached, a new log file is generated. The default size limit is 10485760 (10 MB).
           </p>
           <h4>
-           <a id="_literal_logging_files_keepfiles_literal">
+           <a id="_logging_files_keepfiles">
            </a>
            <code class="literal">
             logging.files.keepfiles
@@ -536,7 +536,7 @@
            options has to be in the range of 2 to 1024 files.
           </p>
           <h4>
-           <a id="_literal_logging_files_permissions_literal">
+           <a id="_logging_files_permissions">
            </a>
            <code class="literal">
             logging.files.permissions
@@ -575,7 +575,7 @@
            </ul>
           </div>
           <h4>
-           <a id="_literal_logging_files_interval_literal">
+           <a id="_logging_files_interval">
            </a>
            <code class="literal">
             logging.files.interval
@@ -588,7 +588,7 @@
            Enable log file rotation on time intervals in addition to size-based rotation. Intervals must be at least 1s. Values of 1m, 1h, 24h, 7*24h, 30*24h, and 365*24h are boundary-aligned with minutes, hours, days, weeks, months, and years as reported by the local system clock. All other intervals are calculated from the unix epoch. Defaults to disabled.
           </p>
           <h4>
-           <a id="_literal_logging_files_rotateonstartup_literal">
+           <a id="_logging_files_rotateonstartup">
            </a>
            <code class="literal">
             logging.files.rotateonstartup
@@ -601,7 +601,7 @@
            If the log file already exists on startup, immediately rotate it and start writing to a new file instead of appending to the existing one. Defaults to true.
           </p>
           <h4>
-           <a id="_literal_logging_json_literal">
+           <a id="_logging_json">
            </a>
            <code class="literal">
             logging.json
@@ -614,7 +614,7 @@
            When true, logs messages in JSON format. The default is false.
           </p>
           <h4>
-           <a id="_literal_logging_files_redirect_stderr_literal_phrase_role_experimental_phrase">
+           <a id="_logging_files_redirect_stderr">
            </a>
            <code class="literal">
             logging.files.redirect_stderr
@@ -681,17 +681,17 @@
           </p>
           <p>
            <code class="literal">
-            2017-12-17T18:54:16.241-0500   INFO    logp/core_test.go:13    unnamed global logger
-           </code>
-          </p>
-          <p>
-           <code class="literal">
-            2017-12-17T18:54:16.242-0500   INFO    [example]       logp/core_test.go:16    some message
-           </code>
-          </p>
-          <p>
-           <code class="literal">
-            2017-12-17T18:54:16.242-0500   INFO    [example]       logp/core_test.go:19    some message    {"x": 1}
+            2017-12-17T18:54:16.241-0500	INFO	logp/core_test.go:13	unnamed global logger
+           </code>
+          </p>
+          <p>
+           <code class="literal">
+            2017-12-17T18:54:16.242-0500	INFO	[example]	logp/core_test.go:16	some message
+           </code>
+          </p>
+          <p>
+           <code class="literal">
+            2017-12-17T18:54:16.242-0500	INFO	[example]	logp/core_test.go:19	some message	{"x": 1}
            </code>
           </p>
          </div>
html/en/beats/journalbeat/7.4/configuration-monitor.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/configuration-monitor.html
@@ -180,7 +180,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_monitoring_enabled_literal">
+               <a id="_monitoring_enabled">
                </a>
                <code class="literal">
                 monitoring.enabled
@@ -216,7 +216,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_monitoring_elasticsearch_literal">
+               <a id="_monitoring_elasticsearch">
                </a>
                <code class="literal">
                 monitoring.elasticsearch
@@ -237,7 +237,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_bulk_max_size_literal_6">
+               <a id="_bulk_max_size_6">
                </a>
                <code class="literal">
                 bulk_max_size
@@ -266,7 +266,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_backoff_init_literal_4">
+               <a id="_backoff_init_4">
                </a>
                <code class="literal">
                 backoff.init
@@ -295,7 +295,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_backoff_max_literal_4">
+               <a id="_backoff_max_4">
                </a>
                <code class="literal">
                 backoff.max
@@ -316,7 +316,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_compression_level_literal_4">
+               <a id="_compression_level_4">
                </a>
                <code class="literal">
                 compression_level
@@ -353,7 +353,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_headers_literal_2">
+               <a id="_headers_2">
                </a>
                <code class="literal">
                 headers
@@ -378,7 +378,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_hosts_literal_3">
+               <a id="_hosts_3">
                </a>
                <code class="literal">
                 hosts
@@ -403,7 +403,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_max_retries_literal_5">
+               <a id="_max_retries_5">
                </a>
                <code class="literal">
                 max_retries
@@ -432,7 +432,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_parameters_literal_2">
+               <a id="_parameters_2">
                </a>
                <code class="literal">
                 parameters
@@ -453,7 +453,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_password_literal_4">
+               <a id="_password_4">
                </a>
                <code class="literal">
                 password
@@ -474,7 +474,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_metrics_period_literal">
+               <a id="_metrics_period">
                </a>
                <code class="literal">
                 metrics.period
@@ -495,7 +495,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_state_period_literal">
+               <a id="_state_period">
                </a>
                <code class="literal">
                 state.period
@@ -516,7 +516,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_protocol_literal">
+               <a id="_protocol">
                </a>
                <code class="literal">
                 protocol
@@ -553,7 +553,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_proxy_url_literal_4">
+               <a id="_proxy_url_4">
                </a>
                <code class="literal">
                 proxy_url
@@ -578,7 +578,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_timeout_literal_5">
+               <a id="_timeout_5">
                </a>
                <code class="literal">
                 timeout
@@ -603,7 +603,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_ssl_literal_5">
+               <a id="_ssl_5">
                </a>
                <code class="literal">
                 ssl
@@ -634,7 +634,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_username_literal_3">
+               <a id="_username_3">
                </a>
                <code class="literal">
                 username
html/en/beats/journalbeat/7.4/configuration-output-codec.html
html/en/beats/journalbeat/7.4/configuration-path.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/configuration-path.html
@@ -207,7 +207,7 @@
            config file:
           </p>
           <h4>
-           <a id="_literal_home_literal">
+           <a id="_home">
            </a>
            <code class="literal">
             home
@@ -226,7 +226,7 @@
            <pre class="programlisting prettyprint lang-yaml">path.home: /usr/share/beats</pre>
           </div>
           <h4>
-           <a id="_literal_config_literal">
+           <a id="_config">
            </a>
            <code class="literal">
             config
@@ -245,7 +245,7 @@
            <pre class="programlisting prettyprint lang-yaml">path.config: /usr/share/beats/config</pre>
           </div>
           <h4>
-           <a id="_literal_data_literal">
+           <a id="_data">
            </a>
            <code class="literal">
             data
@@ -282,7 +282,7 @@
            </div>
           </div>
           <h4>
-           <a id="_literal_logs_literal">
+           <a id="_logs">
            </a>
            <code class="literal">
             logs
html/en/beats/journalbeat/7.4/configuration-ssl.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/configuration-ssl.html
@@ -224,7 +224,7 @@
            config file:
           </p>
           <h4>
-           <a id="_literal_enabled_literal_7">
+           <a id="_enabled_7">
            </a>
            <code class="literal">
             enabled
@@ -271,7 +271,7 @@
            </div>
           </div>
           <h4>
-           <a id="_literal_certificate_authorities_literal">
+           <a id="_certificate_authorities">
            </a>
            <code class="literal">
             certificate_authorities
@@ -329,7 +329,7 @@
            is specified.
           </p>
           <h4>
-           <a id="_literal_key_passphrase_literal">
+           <a id="_key_passphrase">
            </a>
            <code class="literal">
             key_passphrase
@@ -346,7 +346,7 @@
            file.
           </p>
           <h4>
-           <a id="_literal_supported_protocols_literal">
+           <a id="_supported_protocols">
            </a>
            <code class="literal">
             supported_protocols
@@ -386,7 +386,7 @@
            .
           </p>
           <h4>
-           <a id="_literal_verification_mode_literal">
+           <a id="_verification_mode">
            </a>
            <code class="literal">
             verification_mode
@@ -422,7 +422,7 @@
            .
           </p>
           <h4>
-           <a id="_literal_cipher_suites_literal">
+           <a id="_cipher_suites">
            </a>
            <code class="literal">
             cipher_suites
@@ -542,7 +542,7 @@
            </ul>
           </div>
           <h4>
-           <a id="_literal_curve_types_literal">
+           <a id="_curve_types">
            </a>
            <code class="literal">
             curve_types
@@ -574,7 +574,7 @@
            </ul>
           </div>
           <h4>
-           <a id="_literal_renegotiation_literal">
+           <a id="_renegotiation">
            </a>
            <code class="literal">
             renegotiation
html/en/beats/journalbeat/7.4/configuration-template.html
html/en/beats/journalbeat/7.4/configure-cloud-id.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/configure-cloud-id.html
@@ -188,7 +188,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_cloud_id_literal">
+               <a id="_cloud_id">
                </a>
                <code class="literal">
                 cloud.id
@@ -217,7 +217,7 @@
             <div>
              <div>
               <h3 class="title">
-               <a id="_literal_cloud_auth_literal">
+               <a id="_cloud_auth">
                </a>
                <code class="literal">
                 cloud.auth
html/en/beats/journalbeat/7.4/configuring-howto-journalbeat.html
html/en/beats/journalbeat/7.4/configuring-ingest-node.html
html/en/beats/journalbeat/7.4/configuring-internal-queue.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/configuring-internal-queue.html
@@ -250,7 +250,7 @@
            config file:
           </p>
           <h5>
-           <a id="_literal_events_literal">
+           <a id="_events">
            </a>
            <code class="literal">
             events
@@ -266,7 +266,7 @@
            The default value is 4096 events.
           </p>
           <h5>
-           <a id="_literal_flush_min_events_literal">
+           <a id="_flush_min_events">
            </a>
            <code class="literal">
             flush.min_events
@@ -282,7 +282,7 @@
            The default value is 2048.
           </p>
           <h5>
-           <a id="_literal_flush_timeout_literal">
+           <a id="_flush_timeout">
            </a>
            <code class="literal">
             flush.timeout
@@ -376,7 +376,7 @@
            config file:
           </p>
           <h5>
-           <a id="_literal_file_path_literal">
+           <a id="_file_path">
            </a>
            <code class="literal">
             file.path
@@ -392,7 +392,7 @@
            The default value is "${path.data}/spool.dat".
           </p>
           <h5>
-           <a id="_literal_file_permissions_literal">
+           <a id="_file_permissions">
            </a>
            <code class="literal">
             file.permissions
@@ -412,7 +412,7 @@
            The default value is 0600.
           </p>
           <h5>
-           <a id="_literal_file_size_literal">
+           <a id="_file_size">
            </a>
            <code class="literal">
             file.size
@@ -448,7 +448,7 @@
            </div>
           </div>
           <h5>
-           <a id="_literal_file_page_size_literal">
+           <a id="_file_page_size">
            </a>
            <code class="literal">
             file.page_size
@@ -495,7 +495,7 @@
            </div>
           </div>
           <h5>
-           <a id="_literal_file_prealloc_literal">
+           <a id="_file_prealloc">
            </a>
            <code class="literal">
             file.prealloc
@@ -542,7 +542,7 @@
            .
           </p>
           <h5>
-           <a id="_literal_write_buffer_size_literal">
+           <a id="_write_buffer_size">
            </a>
            <code class="literal">
             write.buffer_size
@@ -561,7 +561,7 @@
            The default value is 1MiB.
           </p>
           <h5>
-           <a id="_literal_write_codec_literal">
+           <a id="_write_codec">
            </a>
            <code class="literal">
             write.codec
@@ -589,7 +589,7 @@
            .
           </p>
           <h5>
-           <a id="_literal_write_flush_timeout_literal">
+           <a id="_write_flush_timeout">
            </a>
            <code class="literal">
             write.flush.timeout
@@ -613,7 +613,7 @@
            The default value is 1s.
           </p>
           <h5>
-           <a id="_literal_write_flush_events_literal">
+           <a id="_write_flush_events">
            </a>
            <code class="literal">
             write.flush.events
@@ -629,7 +629,7 @@
            The default value is 16384.
           </p>
           <h5>
-           <a id="_literal_read_flush_timeout_literal">
+           <a id="_read_flush_timeout">
            </a>
            <code class="literal">
             read.flush.timeout
html/en/beats/journalbeat/7.4/configuring-output.html
html/en/beats/journalbeat/7.4/configuring-ssl-logstash.html
html/en/beats/journalbeat/7.4/connection-problem.html
html/en/beats/journalbeat/7.4/console-output.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/console-output.html
@@ -201,7 +201,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_pretty_literal">
+                <a id="_pretty">
                 </a>
                 <code class="literal">
                  pretty
@@ -226,7 +226,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_codec_literal_4">
+                <a id="_codec_4">
                 </a>
                 <code class="literal">
                  codec
@@ -262,7 +262,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal_6">
+                <a id="_enabled_6">
                 </a>
                 <code class="literal">
                  enabled
@@ -286,7 +286,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_max_size_literal_5">
+                <a id="_bulk_max_size_5">
                 </a>
                 <code class="literal">
                  bulk_max_size
html/en/beats/journalbeat/7.4/convert.html
html/en/beats/journalbeat/7.4/decode-base64-field.html
html/en/beats/journalbeat/7.4/decode-csv-fields.html
html/en/beats/journalbeat/7.4/decode-json-fields.html
html/en/beats/journalbeat/7.4/decompress-gzip-field.html
html/en/beats/journalbeat/7.4/defining-processors.html
html/en/beats/journalbeat/7.4/diff-logstash-beats.html
html/en/beats/journalbeat/7.4/directory-layout.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/directory-layout.html
@@ -201,10 +201,6 @@
                </p>
               </td>
               <td align="left" valign="top">
-               <p>
-                <code class="literal">
-                </code>
-               </p>
               </td>
               <td align="left" valign="top">
                <p>
@@ -237,10 +233,6 @@
                </p>
               </td>
               <td align="left" valign="top">
-               <p>
-                <code class="literal">
-                </code>
-               </p>
               </td>
              </tr>
              <tr>
html/en/beats/journalbeat/7.4/dissect.html
html/en/beats/journalbeat/7.4/drop-event.html
html/en/beats/journalbeat/7.4/drop-fields.html
html/en/beats/journalbeat/7.4/elasticsearch-output.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/elasticsearch-output.html
@@ -284,7 +284,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal">
+                <a id="_enabled">
                 </a>
                 <code class="literal">
                  enabled
@@ -408,7 +408,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_compression_level_literal">
+                <a id="_compression_level">
                 </a>
                 <code class="literal">
                  compression_level
@@ -435,7 +435,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_escape_html_literal">
+                <a id="_escape_html">
                 </a>
                 <code class="literal">
                  escape_html
@@ -467,7 +467,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_worker_literal">
+                <a id="_worker">
                 </a>
                 <code class="literal">
                  worker
@@ -491,7 +491,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_username_literal">
+                <a id="_username">
                 </a>
                 <code class="literal">
                  username
@@ -512,7 +512,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_password_literal">
+                <a id="_password">
                 </a>
                 <code class="literal">
                  password
@@ -533,7 +533,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_parameters_literal">
+                <a id="_parameters">
                 </a>
                 <code class="literal">
                  parameters
@@ -618,7 +618,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_headers_literal">
+                <a id="_headers">
                 </a>
                 <code class="literal">
                  headers
@@ -646,7 +646,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_proxy_url_literal">
+                <a id="_proxy_url">
                 </a>
                 <code class="literal">
                  proxy_url
@@ -1282,7 +1282,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_max_retries_literal">
+                <a id="_max_retries">
                 </a>
                 <code class="literal">
                  max_retries
@@ -1313,7 +1313,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_max_size_literal">
+                <a id="_bulk_max_size">
                 </a>
                 <code class="literal">
                  bulk_max_size
@@ -1351,7 +1351,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_init_literal">
+                <a id="_backoff_init">
                 </a>
                 <code class="literal">
                  backoff.init
@@ -1380,7 +1380,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_max_literal">
+                <a id="_backoff_max">
                 </a>
                 <code class="literal">
                  backoff.max
@@ -1401,7 +1401,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_timeout_literal">
+                <a id="_timeout">
                 </a>
                 <code class="literal">
                  timeout
@@ -1422,7 +1422,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_ssl_literal">
+                <a id="_ssl">
                 </a>
                 <code class="literal">
                  ssl
html/en/beats/journalbeat/7.4/enable-journalbeat-debugging.html
html/en/beats/journalbeat/7.4/error-found-unexpected-character.html
html/en/beats/journalbeat/7.4/error-loading-config.html
html/en/beats/journalbeat/7.4/exported-fields-beat-common.html
html/en/beats/journalbeat/7.4/exported-fields-cloud.html
html/en/beats/journalbeat/7.4/exported-fields-common.html
html/en/beats/journalbeat/7.4/exported-fields-docker-processor.html
html/en/beats/journalbeat/7.4/exported-fields-ecs.html
html/en/beats/journalbeat/7.4/exported-fields-host-processor.html
html/en/beats/journalbeat/7.4/exported-fields-jolokia-autodiscover.html
html/en/beats/journalbeat/7.4/exported-fields-kubernetes-processor.html
html/en/beats/journalbeat/7.4/exported-fields-process.html
html/en/beats/journalbeat/7.4/exported-fields.html
html/en/beats/journalbeat/7.4/extract-array.html
html/en/beats/journalbeat/7.4/faq.html
html/en/beats/journalbeat/7.4/feature-roles.html
html/en/beats/journalbeat/7.4/file-output.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/file-output.html
@@ -205,7 +205,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal_5">
+                <a id="_enabled_5">
                 </a>
                 <code class="literal">
                  enabled
@@ -250,7 +250,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_filename_literal">
+                <a id="_filename">
                 </a>
                 <code class="literal">
                  filename
@@ -271,7 +271,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_rotate_every_kb_literal">
+                <a id="_rotate_every_kb">
                 </a>
                 <code class="literal">
                  rotate_every_kb
@@ -292,7 +292,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_number_of_files_literal">
+                <a id="_number_of_files">
                 </a>
                 <code class="literal">
                  number_of_files
@@ -319,7 +319,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_permissions_literal">
+                <a id="_permissions">
                 </a>
                 <code class="literal">
                  permissions
@@ -340,7 +340,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_codec_literal_3">
+                <a id="_codec_3">
                 </a>
                 <code class="literal">
                  codec
html/en/beats/journalbeat/7.4/filtering-and-enhancing-data.html
html/en/beats/journalbeat/7.4/getting-help.html
html/en/beats/journalbeat/7.4/http-endpoint.html
html/en/beats/journalbeat/7.4/ilm.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/ilm.html
@@ -274,7 +274,7 @@
           <p>
            The index lifecycle write alias name. The default is
            <code class="literal">
-            journalbeat-{agent.version}
+            journalbeat-%{agent.version}
            </code>
            . Setting this option changes the alias name.
           </p>
@@ -341,7 +341,7 @@
           <p>
            The name to use for the lifecycle policy. The default is
            <code class="literal">
-            journalbeat-{agent.version}
+            journalbeat-%{[agent.version]}
            </code>
            .
           </p>
html/en/beats/journalbeat/7.4/include-fields.html
html/en/beats/journalbeat/7.4/index.html
html/en/beats/journalbeat/7.4/journalbeat-configuration.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/journalbeat-configuration.html
@@ -489,17 +489,17 @@
                      </a>
                      or equivalent privileges.
                     </p>
-                    <p>
-                     For more information, see
-                     <a class="xref" href="securing-journalbeat.html" title="Securing Journalbeat">
-                      Securing Journalbeat
-                     </a>
-                     .
-                    </p>
                    </td>
                   </tr>
                  </table>
                 </div>
+                <p class="simpara">
+                 For more information, see
+                 <a class="xref" href="securing-journalbeat.html" title="Securing Journalbeat">
+                  Securing Journalbeat
+                 </a>
+                 .
+                </p>
                </li>
               </ul>
              </div>
html/en/beats/journalbeat/7.4/journalbeat-geoip.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/journalbeat-geoip.html
@@ -222,8 +222,8 @@
               </code>
               processors to add location information to the event. For example, you can use the Console in Kibana to create the following pipeline:
              </p>
-             <div class="pre_wrapper lang-json">
-              <pre class="programlisting prettyprint lang-json">PUT _ingest/pipeline/geoip-info
+             <div class="pre_wrapper lang-console">
+              <pre class="programlisting prettyprint lang-console">PUT _ingest/pipeline/geoip-info
 {
   "description": "Add geoip info",
   "processors": [
html/en/beats/journalbeat/7.4/journalbeat-getting-started.html
html/en/beats/journalbeat/7.4/journalbeat-installation.html
html/en/beats/journalbeat/7.4/journalbeat-overview.html
html/en/beats/journalbeat/7.4/journalbeat-reference-yml.html
html/en/beats/journalbeat/7.4/journalbeat-starting.html
html/en/beats/journalbeat/7.4/journalbeat-template.html
html/en/beats/journalbeat/7.4/kafka-output.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/kafka-output.html
@@ -255,7 +255,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal_3">
+                <a id="_enabled_3">
                 </a>
                 <code class="literal">
                  enabled
@@ -283,7 +283,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_hosts_literal">
+                <a id="_hosts">
                 </a>
                 <code class="literal">
                  hosts
@@ -304,7 +304,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_version_literal">
+                <a id="_version">
                 </a>
                 <code class="literal">
                  version
@@ -346,7 +346,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_username_literal_2">
+                <a id="_username_2">
                 </a>
                 <code class="literal">
                  username
@@ -367,7 +367,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_password_literal_2">
+                <a id="_password_2">
                 </a>
                 <code class="literal">
                  password
@@ -592,7 +592,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_key_literal">
+                <a id="_key">
                 </a>
                 <code class="literal">
                  key
@@ -613,7 +613,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_partition_literal">
+                <a id="_partition">
                 </a>
                 <code class="literal">
                  partition
@@ -719,7 +719,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_client_id_literal">
+                <a id="_client_id">
                 </a>
                 <code class="literal">
                  client_id
@@ -740,7 +740,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_worker_literal_3">
+                <a id="_worker_3">
                 </a>
                 <code class="literal">
                  worker
@@ -761,7 +761,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_codec_literal">
+                <a id="_codec">
                 </a>
                 <code class="literal">
                  codec
@@ -793,7 +793,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_metadata_literal">
+                <a id="_metadata">
                 </a>
                 <code class="literal">
                  metadata
@@ -882,7 +882,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_max_retries_literal_3">
+                <a id="_max_retries_3">
                 </a>
                 <code class="literal">
                  max_retries
@@ -913,7 +913,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_max_size_literal_3">
+                <a id="_bulk_max_size_3">
                 </a>
                 <code class="literal">
                  bulk_max_size
@@ -934,7 +934,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_flush_frequency_literal">
+                <a id="_bulk_flush_frequency">
                 </a>
                 <code class="literal">
                  bulk_flush_frequency
@@ -955,7 +955,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_timeout_literal_3">
+                <a id="_timeout_3">
                 </a>
                 <code class="literal">
                  timeout
@@ -976,7 +976,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_broker_timeout_literal">
+                <a id="_broker_timeout">
                 </a>
                 <code class="literal">
                  broker_timeout
@@ -997,7 +997,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_channel_buffer_size_literal">
+                <a id="_channel_buffer_size">
                 </a>
                 <code class="literal">
                  channel_buffer_size
@@ -1018,7 +1018,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_keep_alive_literal">
+                <a id="_keep_alive">
                 </a>
                 <code class="literal">
                  keep_alive
@@ -1039,7 +1039,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_compression_literal">
+                <a id="_compression">
                 </a>
                 <code class="literal">
                  compression
@@ -1080,7 +1080,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_compression_level_literal_3">
+                <a id="_compression_level_3">
                 </a>
                 <code class="literal">
                  compression_level
@@ -1132,7 +1132,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_required_acks_literal">
+                <a id="_required_acks">
                 </a>
                 <code class="literal">
                  required_acks
@@ -1156,7 +1156,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_ssl_literal_3">
+                <a id="_ssl_3">
                 </a>
                 <code class="literal">
                  ssl
html/en/beats/journalbeat/7.4/keystore.html
html/en/beats/journalbeat/7.4/linux-seccomp.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/linux-seccomp.html
@@ -347,6 +347,12 @@
             <dd>
              A list of system call names. The system call name must exist for the runtime architecture, otherwise an error will be logged and the filter will not be installed. At least one system call must be defined.
             </dd>
+           </dl>
+          </div>
+          <div class="variablelist">
+           <a id="seccomp-policy-config-action">
+           </a>
+           <dl class="variablelist">
             <dt>
              <span class="term">
               <span class="strong strong">
@@ -367,8 +373,6 @@
               is executed. This is required. These are the available action values. The actions that are available depend on the kernel version.
              </p>
              <div class="itemizedlist">
-              <a id="seccomp-policy-config-action">
-              </a>
               <ul class="itemizedlist" type="disc">
                <li class="listitem">
                 <code class="literal">
html/en/beats/journalbeat/7.4/logstash-output.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/logstash-output.html
@@ -171,6 +171,9 @@
             <img alt="Important" src="images/icons/important.png"/>
            </div>
            <div class="admon_content">
+            <h3>
+             Prerequisite
+            </h3>
             <p>
              To send events to Logstash, you also need to create a Logstash configuration pipeline that listens for incoming Beats connections and indexes the received events into Elasticsearch. For more information, see the section about
              <a class="ulink" href="https://www.elastic.co/guide/en/elastic-stack-get-started/7.4/get-started-elastic-stack.html#logstash-setup" target="_top">
@@ -509,7 +512,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal_2">
+                <a id="_enabled_2">
                 </a>
                 <code class="literal">
                  enabled
@@ -557,7 +560,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_compression_level_literal_2">
+                <a id="_compression_level_2">
                 </a>
                 <code class="literal">
                  compression_level
@@ -584,7 +587,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_escape_html_literal_2">
+                <a id="_escape_html_2">
                 </a>
                 <code class="literal">
                  escape_html
@@ -616,7 +619,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_worker_literal_2">
+                <a id="_worker_2">
                 </a>
                 <code class="literal">
                  worker
@@ -664,7 +667,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_ttl_literal">
+                <a id="_ttl">
                 </a>
                 <code class="literal">
                  ttl
@@ -698,7 +701,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_pipelining_literal">
+                <a id="_pipelining">
                 </a>
                 <code class="literal">
                  pipelining
@@ -723,7 +726,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_proxy_url_literal_2">
+                <a id="_proxy_url_2">
                 </a>
                 <code class="literal">
                  proxy_url
@@ -823,7 +826,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_ssl_literal_2">
+                <a id="_ssl_2">
                 </a>
                 <code class="literal">
                  ssl
@@ -854,7 +857,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_timeout_literal_2">
+                <a id="_timeout_2">
                 </a>
                 <code class="literal">
                  timeout
@@ -875,7 +878,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_max_retries_literal_2">
+                <a id="_max_retries_2">
                 </a>
                 <code class="literal">
                  max_retries
@@ -906,7 +909,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_max_size_literal_2">
+                <a id="_bulk_max_size_2">
                 </a>
                 <code class="literal">
                  bulk_max_size
@@ -944,7 +947,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_slow_start_literal">
+                <a id="_slow_start">
                 </a>
                 <code class="literal">
                  slow_start
@@ -976,7 +979,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_init_literal_2">
+                <a id="_backoff_init_2">
                 </a>
                 <code class="literal">
                  backoff.init
@@ -1005,7 +1008,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_max_literal_2">
+                <a id="_backoff_max_2">
                 </a>
                 <code class="literal">
                  backoff.max
html/en/beats/journalbeat/7.4/metadata-missing.html
html/en/beats/journalbeat/7.4/monitoring-internal-collection.html
html/en/beats/journalbeat/7.4/monitoring-metricbeat-collection.html
html/en/beats/journalbeat/7.4/monitoring-shows-fewer-than-expected-beats.html
html/en/beats/journalbeat/7.4/monitoring.html
html/en/beats/journalbeat/7.4/processor-dns.html
html/en/beats/journalbeat/7.4/processor-registered-domain.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/processor-registered-domain.html
@@ -205,7 +205,7 @@
            processor has the following configuration settings:
           </p>
           <div class="table">
-           <a id="id-1.5.9.31.6">
+           <a id="id-1.5.9.26.6">
            </a>
            <p class="title">
             <strong>
html/en/beats/journalbeat/7.4/processor-script.html
html/en/beats/journalbeat/7.4/processor-timestamp.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/processor-timestamp.html
@@ -229,7 +229,7 @@
            is added to the time value.
           </p>
           <div class="table">
-           <a id="id-1.5.9.41.11">
+           <a id="id-1.5.9.36.11">
            </a>
            <p class="title">
             <strong>
html/en/beats/journalbeat/7.4/redis-output.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/redis-output.html
@@ -231,7 +231,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_enabled_literal_4">
+                <a id="_enabled_4">
                 </a>
                 <code class="literal">
                  enabled
@@ -255,7 +255,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_hosts_literal_2">
+                <a id="_hosts_2">
                 </a>
                 <code class="literal">
                  hosts
@@ -296,7 +296,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_index_literal">
+                <a id="_index">
                 </a>
                 <code class="literal">
                  index
@@ -525,7 +525,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_password_literal_3">
+                <a id="_password_3">
                 </a>
                 <code class="literal">
                  password
@@ -546,7 +546,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_db_literal">
+                <a id="_db">
                 </a>
                 <code class="literal">
                  db
@@ -567,7 +567,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_datatype_literal">
+                <a id="_datatype">
                 </a>
                 <code class="literal">
                  datatype
@@ -612,7 +612,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_codec_literal_2">
+                <a id="_codec_2">
                 </a>
                 <code class="literal">
                  codec
@@ -644,7 +644,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_host_topology_literal">
+                <a id="_host_topology">
                 </a>
                 <code class="literal">
                  host_topology
@@ -656,16 +656,18 @@
               </div>
              </div>
             </div>
-            <div class="warning admon">
-             <div class="icon">
-              <img alt="Warning" src="images/icons/warning.png"/>
-             </div>
-             <div class="admon_content">
-              <p>
+            <p>
+             <span class="Admonishment">
+              [
+              <span class="Admonishment-version u-mono u-strikethrough">
+               5.0.0
+              </span>
+              ]
+              <span class="Admonishment-detail">
                Deprecated in 5.0.0.
-              </p>
-             </div>
-            </div>
+              </span>
+             </span>
+            </p>
             <p>
              The Redis host to connect to when using topology map support. Topology map support is disabled if this option is not set.
             </p>
@@ -675,7 +677,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_password_topology_literal">
+                <a id="_password_topology">
                 </a>
                 <code class="literal">
                  password_topology
@@ -687,16 +689,18 @@
               </div>
              </div>
             </div>
-            <div class="warning admon">
-             <div class="icon">
-              <img alt="Warning" src="images/icons/warning.png"/>
-             </div>
-             <div class="admon_content">
-              <p>
+            <p>
+             <span class="Admonishment">
+              [
+              <span class="Admonishment-version u-mono u-strikethrough">
+               5.0.0
+              </span>
+              ]
+              <span class="Admonishment-detail">
                Deprecated in 5.0.0.
-              </p>
-             </div>
-            </div>
+              </span>
+             </span>
+            </p>
             <p>
              The password to use for authenticating with the Redis topology server. The default is no authentication.
             </p>
@@ -706,7 +710,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_db_topology_literal">
+                <a id="_db_topology">
                 </a>
                 <code class="literal">
                  db_topology
@@ -718,16 +722,18 @@
               </div>
              </div>
             </div>
-            <div class="warning admon">
-             <div class="icon">
-              <img alt="Warning" src="images/icons/warning.png"/>
-             </div>
-             <div class="admon_content">
-              <p>
+            <p>
+             <span class="Admonishment">
+              [
+              <span class="Admonishment-version u-mono u-strikethrough">
+               5.0.0
+              </span>
+              ]
+              <span class="Admonishment-detail">
                Deprecated in 5.0.0.
-              </p>
-             </div>
-            </div>
+              </span>
+             </span>
+            </p>
             <p>
              The Redis database number where the topology information is stored. The default is 1.
             </p>
@@ -737,7 +743,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_worker_literal_4">
+                <a id="_worker_4">
                 </a>
                 <code class="literal">
                  worker
@@ -762,7 +768,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_loadbalance_literal">
+                <a id="_loadbalance">
                 </a>
                 <code class="literal">
                  loadbalance
@@ -783,7 +789,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_timeout_literal_4">
+                <a id="_timeout_4">
                 </a>
                 <code class="literal">
                  timeout
@@ -804,7 +810,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_init_literal_3">
+                <a id="_backoff_init_3">
                 </a>
                 <code class="literal">
                  backoff.init
@@ -833,7 +839,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_backoff_max_literal_3">
+                <a id="_backoff_max_3">
                 </a>
                 <code class="literal">
                  backoff.max
@@ -854,7 +860,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_max_retries_literal_4">
+                <a id="_max_retries_4">
                 </a>
                 <code class="literal">
                  max_retries
@@ -885,7 +891,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_bulk_max_size_literal_4">
+                <a id="_bulk_max_size_4">
                 </a>
                 <code class="literal">
                  bulk_max_size
@@ -923,7 +929,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_ssl_literal_4">
+                <a id="_ssl_4">
                 </a>
                 <code class="literal">
                  ssl
@@ -954,7 +960,7 @@
              <div>
               <div>
                <h4 class="title">
-                <a id="_literal_proxy_url_literal_3">
+                <a id="_proxy_url_3">
                 </a>
                 <code class="literal">
                  proxy_url
html/en/beats/journalbeat/7.4/regexp-support.html
html/en/beats/journalbeat/7.4/rename-fields.html
html/en/beats/journalbeat/7.4/running-on-docker.html
html/en/beats/journalbeat/7.4/running-with-systemd.html
html/en/beats/journalbeat/7.4/securing-beats.html
html/en/beats/journalbeat/7.4/securing-communication-elasticsearch.html
html/en/beats/journalbeat/7.4/securing-journalbeat.html
html/en/beats/journalbeat/7.4/setting-up-and-running.html
html/en/beats/journalbeat/7.4/setup-kibana-endpoint.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/setup-kibana-endpoint.html
@@ -197,7 +197,7 @@
            config file:
           </p>
           <h4>
-           <a id="_literal_setup_kibana_host_literal">
+           <a id="_setup_kibana_host">
            </a>
            <code class="literal">
             setup.kibana.host
@@ -320,7 +320,7 @@
 setup.kibana.path: /kibana</pre>
           </div>
           <h4>
-           <a id="_literal_setup_kibana_username_literal">
+           <a id="_setup_kibana_username">
            </a>
            <code class="literal">
             setup.kibana.username
@@ -337,7 +337,7 @@
            specified for the Elasticsearch output.
           </p>
           <h4>
-           <a id="_literal_setup_kibana_password_literal">
+           <a id="_setup_kibana_password">
            </a>
            <code class="literal">
             setup.kibana.password
@@ -367,7 +367,7 @@
            An HTTP path prefix that is prepended to the HTTP API calls. This is useful for the cases where Kibana listens behind an HTTP reverse proxy that exports the API under a custom prefix.
           </p>
           <h4>
-           <a id="_literal_setup_kibana_ssl_enabled_literal">
+           <a id="_setup_kibana_ssl_enabled">
            </a>
            <code class="literal">
             setup.kibana.ssl.enabled
html/en/beats/journalbeat/7.4/setup-repositories.html
html/en/beats/journalbeat/7.4/shutdown.html
html/en/beats/journalbeat/7.4/ssl-client-fails.html
html/en/beats/journalbeat/7.4/toc.html
html/en/beats/journalbeat/7.4/troubleshooting.html
html/en/beats/journalbeat/7.4/using-environ-vars.html
--- /dev/fd/63
+++ html/en/beats/journalbeat/7.4/using-environ-vars.html
@@ -463,7 +463,7 @@
              <code class="literal">
               \
              </code>
-             ) will be interpreted as an escape character. :standalone!:
+             ) will be interpreted as an escape character.
             </p>
            </div>
           </div>
html/en/beats/journalbeat/7.4/view-kibana-dashboards.html
html/en/beats/journalbeat/7.4/yaml-tips.html
